config/gulp/javascript.js

```diff
@@ -0,0 +1,55 @@
+var gulp = require( 'gulp' );
+var gutil = require( 'gulp-util' );
+var dutil = require( './doc-util' );
+var browserify = require( 'browserify' );
+var buffer = require( 'vinyl-buffer' );
+var source = require( 'vinyl-source-stream' );
+var uglify = require( 'gulp-uglify' );
+var sourcemaps = require( 'gulp-sourcemaps' );
+var rename = require( 'gulp-rename' );
+var assert = require( 'gulp-if' );
+var linter = require( 'gulp-eslint' );
+var task = /([\w\d-_]+)\.js$/.exec( __filename )[ 1 ];
+
+gulp.task( 'eslint', function ( done ) {
+
+  if ( ! cFlags.test ) {
```

Comment:
I'm okay with updating the JS to any style. I'm using the [jQuery style of liberal spacing for readability](http://contribute.jquery.org/style-guide/js/#spacing). But I am open to anything as long as we can lint for it. I know `eslint` supports [enforcing either style around spacing](http://eslint.org/docs/rules/).